plugins/systemstat.koplugin/main.lua

```diff
@@ -180,37 +184,50 @@ function SystemStat:appendProcessInfo()
     end
 
     if #t < 23 then return end
-    n1 = tonumber(t[23])
-    if n1 ~= nil then
-        self:put({_("  Virtual memory (MB)"), string.format("%.2f", n1 / 1024 / 1024)})
-    end
-
-    if #t < 24 then return end
-    n1 = tonumber(t[24])
-    if n1 ~= nil then
-        self:put({_("  RAM usage (MB)"), string.format("%.2f", n1 / 256)})
+    local virtual_mem = tonumber(t[23])
+    local ram_usage = tonumber(t[24]) -- will give nil if not avail
+
+    if virtual_mem ~= nil then
+        local key = _("  Virtual memory") .. (ram_usage and SEP .._("RAM usage") or "")
```

Comment:
I would put RAM usage (RSS) first - which is the real interesting thing.
And second Virtual memory (VSZ), which might be bigger than the max ram on the device, and it's not really obvious what this is, so it can be confusing.
May be it should just be dropped (unless @NiLuJe or @Frenzie think it's a useful indicator to have ?)

Reply:
Ah, so it was a plot to cause https://github.com/koreader/koreader/pull/8499#issuecomment-984080061 and get rid of VSZ, was it? :-P

Reply:
I'm not really sure if it's useful or mostly just there.

Reply:
I would drop it.

Reply:
Yeah, it's mostly of interest to developers, and we have much better tools to actually monitor that, so +1 on dropping it :).